lonboard/_map.py

```diff
@@ -167,6 +167,44 @@ def __init__(
       [`lonboard.basemap.CartoBasemap.PositronNoLabels`][lonboard.basemap.CartoBasemap.PositronNoLabels]
     """
 
+    custom_attribution = traitlets.Union(
+        [
+            traitlets.Unicode(allow_none=True),
+            traitlets.List(traitlets.Unicode(allow_none=False)),
+        ]
+    ).tag(sync=True)
+    """
+    Custom attribution to display on the map.
+
+    This attribute supports the same format as the `attribution` property in the
+    Maplibre API.
+
+    - Type: `str` or `List[str]`
+    - Default: `None`
+
+    You can provide either a single string or a list of strings for custom attributions.
+    If an attribution value is set in the map style, it will be displayed in addition to
+    this custom attribution.
+
+    **Example:**
+
+        ```py
+        m = Map(
+            layers,
+            custom_attribution="Development Seed"
```

Comment:
Should we maybe mention that this `custom_attribution` string can be HTML? Or should we wait on that until the link is actually clickable with fixed z-index?

Reply:
I believe we should mention it once it is supported properly. Let's keep tracking it on #561.